datadog/resource_datadog_logs_index.go

```diff
@@ -111,6 +127,9 @@ func updateLogsIndexState(d *schema.ResourceData, index *datadogV1.LogsIndex) di
 	if err := d.Set("name", index.GetName()); err != nil {
 		return diag.FromErr(err)
 	}
+	if err := d.Set("disable_daily_limit", !index.HasDailyLimit()); err != nil {
```

Comment:
Why use `HasDailyLimit()` here, and not `GetDisableDailyLimit()` ?

Reply:
Because the `disable_daily_limit` is a field that can be set only on the update request object and is never returned.  See here: https://github.com/DataDog/datadog-api-spec/blob/master/spec/v1/logs_indexes.yaml#L287